versioned/tiered-tests/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2020 Dremio
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.projectnessie</groupId>
+    <artifactId>nessie-versioned</artifactId>
+    <version>0.2.2-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>nessie-versioned-tiered-tests</artifactId>
+
+  <name>Nessie - Versioned - TieredVersionStore Tests</name>
```

Comment:
aren't these really store tests, not TieredVersionStore tests?

I'm also wondering if we should just move these into the spi module... 

Thoughts @laurentgo ?

Reply:
Do you mean under `src/test/java`? Maven does not have different set of dependencies for jars produced by the same module, which might cause issue for other projects which want to reuse the tests, hence why the versioned tests were made a separate module for example

Reply:
They are store tests, so we can put it in nessie-versioned-store-tests. I had patterned it after the spi-tests as a separate package.